lib/model/autocomplete.dart

```diff
@@ -615,6 +605,59 @@ class MentionAutocompleteView extends AutocompleteView<MentionAutocompleteQuery,
     return userAName.compareTo(userBName); // TODO(i18n): add locale-aware sorting
   }
 
+  void computeWildcardMentionResults({
+    required List<MentionAutocompleteResult> results,
+    required bool isComposingChannelMessage,
+  }) {
+    if (query.silent) return;
+
+    bool tryOption(WildcardMentionOption option) {
+      if (query.testWildcardOption(option, localizations: localizations)) {
+        results.add(WildcardMentionAutocompleteResult(wildcardOption: option));
+        return true;
+      }
+      return false;
+    }
+
+    // Only one of the (all, everyone, channel, stream) channel wildcards are
+    // shown.
+    all: {
+      if (tryOption(WildcardMentionOption.all)) break all;
+      if (tryOption(WildcardMentionOption.everyone)) break all;
+      if (isComposingChannelMessage) {
+        final isChannelWildcardAvailable = store.account.zulipFeatureLevel >= 247; // TODO(server-9)
+        if (isChannelWildcardAvailable && tryOption(WildcardMentionOption.channel)) break all;
+        if (tryOption(WildcardMentionOption.stream)) break all;
+      }
+    }
+
+    final isTopicWildcardAvailable = store.account.zulipFeatureLevel >= 224; // TODO(sever-8)
```

Comment:
```suggestion
    final isTopicWildcardAvailable = store.account.zulipFeatureLevel >= 224; // TODO(server-8)
```

The spelling is important for these to serve their purpose of being findable via grep :slightly_smiling_face: 

(there's one other like this)